src/modules.c

```diff
@@ -949,6 +968,47 @@ void ModulesCheckInit(void)
     }
 }
 
+void ModulesInitModuleState(void)
+{
+    for (UInt i = 0; i < NrModules; i++) {
+        StructInitInfo * info = Modules[i].info;
+        if (info->initModuleState) {
+            if (SyDebugLoading) {
+                FPUTS_TO_STDERR("#I  InitModuleState(");
+                FPUTS_TO_STDERR(info->name);
+                FPUTS_TO_STDERR(")\n");
+            }
+            Int ret = info->initModuleState();
+            if (ret) {
+                FPUTS_TO_STDERR("#I  InitModuleState(");
+                FPUTS_TO_STDERR(info->name);
```

Comment:
If this happens, how bad is it? I feel like this message isn't very informative to the user if it ever appeared, and I would if we should SyExit if it does, as something REALLY horrible has happened?

Reply:
I agree that calling `Panic` here (which also exits) would probably be better, *but* if we do that, I'd rather do it simultaneously for the code calling `initLibrary`, `initKernel`, etc., upon which this code is modelled. So if you don't mind, I'd rather merge this first, then follow up with a PR changing this to `Panic`